ClickHouse.Ado/ClickHouseConnection.cs

```diff
@@ -33,8 +33,7 @@ public ClickHouseConnection(string connectionString)
         private Stream _stream;
         /*private BinaryReader _reader;
         private BinaryWriter _writer;*/
-        internal ProtocolFormatter Formatter { get;
-            set; }
+        internal ProtocolFormatter Formatter { get; set; }
```

Comment:
All formatting changes should be reverted due to high possibility of merge conflicts in future